LayoutTests/imported/w3c/web-platform-tests/workers/modules/dedicated-worker-import-referrer-expected.txt

```diff
@@ -11,13 +11,13 @@ PASS Same-origin top-level module script loading with "same-origin" referrer pol
 PASS Same-origin static import with "no-referrer" referrer policy.
 PASS Same-origin static import with "origin" referrer policy.
 PASS Same-origin static import with "same-origin" referrer policy.
-FAIL Cross-origin static import with "no-referrer" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: Type error"
-FAIL Cross-origin static import with "origin" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: Type error"
-FAIL Cross-origin static import with "same-origin" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: Type error"
+FAIL Cross-origin static import with "no-referrer" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: "ERROR" cannot be parsed as a URL."
+FAIL Cross-origin static import with "origin" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: "ERROR" cannot be parsed as a URL."
+FAIL Cross-origin static import with "same-origin" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: "ERROR" cannot be parsed as a URL."
 PASS Same-origin dynamic import with "no-referrer" referrer policy.
 PASS Same-origin dynamic import with "origin" referrer policy.
 PASS Same-origin dynamic import with "same-origin" referrer policy.
-FAIL Cross-origin dynamic import with "no-referrer" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: Type error"
-FAIL Cross-origin dynamic import with "origin" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: Type error"
-FAIL Cross-origin dynamic import with "same-origin" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: Type error"
+FAIL Cross-origin dynamic import with "no-referrer" referrer policy. promise_test: Unhandled rejection with value: object "TypeError: "Import failed: TypeError: Cross-origin script load denied by Cross-Origin Resource Sharing policy." cannot be parsed as a URL."
```

Comment:
This looks weird?

Reply:
Yes it looks funny. 
I don't know why the value of URL is this in that test. 
@nt1m do you know why this is happening.

Reply:
This should probably be investigated, Maybe something about the test framework.